roles/manage-gcp-infra/tasks/configure-registry-object-storage.yaml

```diff
@@ -0,0 +1,67 @@
+---
+- name: retrieve the name of one registry pod
+  shell: oc get pod -l deploymentconfig=docker-registry | grep -m 1 registry | awk '{print $1}'
+  register: pod  
+  
+- name: create a temp directory fror the google templates
+  file:
+    path: /tmp/gcp_templates
+    state: directory
+    mode: 0755  
+  
+- name: retrieve the configuration file
+  shell: oc rsync {{ pod.stdout }}:/config.yml /tmp/gcp_templates -n default
+    
+- name: replace with google object storage configuration 1
+#  shell: "sed -i {{ gcloud_sed_expression1 }} /tmp/gcp_templates/config.yml && sed -i {{ gcloud_sed_expression2 }} /tmp/gcp_templates/config.yml"  
+  lineinfile:
+    path: /tmp/gcp_templates/config.yml
+    regexp: "filesystem"
+    state: present
+    line: "  gcs:" 
+
+- name: replace with google object storage configuration 2
+#  shell: "sed -i {{ gcloud_sed_expression1 }} /tmp/gcp_templates/config.yml && sed -i {{ gcloud_sed_expression2 }} /tmp/gcp_templates/config.yml"  
+  lineinfile:
+    path: /tmp/gcp_templates/config.yml
+    regexp: "rootdirectory"
+    state: present
+    line: "    bucket: openshift-{{ gcloud_env }}-registry"           
+  
+- name: check if configmap exists
+  shell: oc get configmap registry-config -n default | grep registry-config  
+  register: configmap
+  changed_when: false
+  failed_when: not (configmap.rc == 0 or configmap.rc == 1)
+  
+- name: delete configmap if it exists
+  shell: oc delete configmap registry-config -n default
```

Comment:
Can we please use a variable for `registry-config`

Reply:
done